website/docs/language/import/index.mdx

```diff
@@ -131,3 +132,32 @@ import {
   id = "i-abcd1234"
 }
 ```
+
+### Importing multiple resources
```

Comment:
Maybe we should mention that you cannot use the `Generate configuration` capability when we are importing resource using `for_each`

Reply:
We should open an issue for that too. It would be a shame to not support that feature.